config/classifiers.yml

```diff
@@ -105,6 +112,7 @@ classifiers:
           stale_branches_count: 1
           life_period: 1
           last_changed: 1
+          questions_resolved_percent: 0.5
```

Comment:
Why do you think that this metric is about maintenance quality?
I think that metrics gathered from StackOverflow are about interaction of project's community (users of that project).
I thought that maintenance metrics are about code of project and it changes.
For instance, number of users creating pull requests or number of users commenting pull requests vs time to close pull request and number of changes in a pull request...
I just think that ```questions_resolved_percent``` is not always correlate with maintainers work, as most of the time  they should just fix or enhance project itself. But on the other hand users share their knowledge and that shows us interest to a particular project.
So I vote for move it to popularity check.

Also, why don't you include those metrics to maturity metrics? I think they are pretty valuable for that. 
WDYT?